src/i18n/cs_CZ/vehicles.ts

```diff
@@ -47,16 +55,16 @@ export default {
         staff: { min: 2, max: 3 },
         special: 'Požadováno po vybudování 4 požárních stanic',
         icon: 'car-side',
-        possibleBuildings: [],
+        possibleBuildings: [0, 18],
     },
     5: {
         caption: 'RZP',
         color: '#9c1c1c',
         credits: 5000,
         coins: 25,
-        staff: { min: 3, max: 3 },
+        staff: { min: 2, max: 2 },
```

Comment:
This looks curious for me as well.

Reply:
![image](https://github.com/user-attachments/assets/c6601c45-0c2f-4a1f-9b67-aa043c5fb15a)
nop and again this is right :D